src/VisualStudio/Core/Next/Remote/ServiceHubRemoteHostClient.cs

```diff
@@ -264,6 +283,14 @@ private static int ReportDetailInfo(IFaultUtility faultUtility)
 
             try
             {
+                // add service hub process.
+                // we will record dumps for all service hub processes
+                foreach (var p in Process.GetProcessesByName("ServiceHub.RoslynCodeAnalysisService32"))
```

Comment:
:memo: This type of code is very fragile with respect to changes like #21839. How would a user implementing a change like that know to update this code as well?

Not sure we need to change anything right now (unless a shared constant with this name already exists somewhere), but it's definitely something to think about.

Reply:
yep. I know, this is 15.3.x specific though. this code doesn't exist in master.